wormchain/Makefile

```diff
@@ -44,17 +43,13 @@ validators:
 build/wormchaind: cmd/wormchaind/main.go $(GO_FILES)
 	go build -v $(BUILD_FLAGS) -tags ledger -o $@ $<
 
-check-ignite:
-	@ if [ "$(shell ignite version | awk '/Ignite CLI version:/ { print $$4 }')" != "$(IGNITE_EXPECTED_VERSION)" ] ; then echo "Expected ignite version $(IGNITE_EXPECTED_VERSION)" && exit 1 ; fi
-
-proto: check-ignite $(PROTO_FILES)
-	ignite generate proto-go
-	touch proto
+proto: $(PROTO_FILES)
+	docker build --target go-export -f Dockerfile.proto -o type=local,dest=wormchain ..
```

Comment:
`dest=.`, this will result in generated files going into `wormchain/wormchain/x/wormhole/types`